src/plugins/lua/bindings/lua_Constants.cpp

```diff
@@ -0,0 +1,283 @@
+         "Tigerclaw",                  46,
+         "Eaglestrike",                47,
+         "EmoteNod",                   48,
+         "EmoteClap",                  51,
+         "EmoteBlush",                 53,
+         "EmoteSnicker",               54,
+         "EmoteCough",                 55,
+         "EmoteDuck",                  56,
+         "EmoteStare",                 57,
+         "EmoteDance",                 58,
+         "EmoteVeto",                  59,
+         "EmoteGlare",                 60,
+         "EmotePeer",                  61,
+         "EmoteGrovel",                62,
+         "EmoteLaugh",                 63,
+         "EmotePoint",                 64,
+         "EmoteShrug",                 65,
+         "EmoteSalute",                67,
+         "EmoteShiver",                68,
+         "EmoteTap",                   69,
+         "EmoteBow",                   70,
+         "Stand1",                     71,
+         "StandAnimated1",             72,
+         "Sounted",                    74,
+         "Melee3",                     80,
+         "Melee4",                     82,
+         "Walking2",                   104,
+         "Reverse",                    105,
+         "Running2",                   106,
+         "TurnRight",                  107,
+         "TurnLeft",                   108,
+         "BindWound2",                 109,
+         "Stand3",                     110,
+         "StandAnimated3",             111,
+         "Melee5",                     112,
+         "Sitting2",                   118,
+         "SitAnimation",               121,
+         "Ducking2",                   122,
+         "JumpRun2",                   123,
+         "JumpInPlace2",               124,
+         "Crawl",                      125,
+         "TreadWater2",                127,
+         "Swimming2",                  128,
+         "Castin4",                    134,
+         "Casting5",                   135,
+         "YouAreHit",                  144,
+         "Drowning2",                  145,
+         "Dead2",                      146,
+         "Falling2",                   151,
+         "EmoteBow2",                  152,
+         "EmoteWave2",                 154,
+         "Salute2",                    155,
+         "EmoteClap2",                 156,
+         "EmotePoint2",                158,
+         "EmoteSnicker2",              159,
+         "EmoteShrug2",                160,
+         "EmoteCry2",                  162,
+         "EmoteRude2",                 163,
+         "EmoteGrovel2",               165,
+         "EmoteBlush2",                166,
+         "EmoteCough2",                167,
+         "EmoteDuck2",                 168,
+         "EmoteShiver2",               172,
+         "EmoteTap2",                  174,
+         "EmoteGlare2",                177,
+         "EmoteStare2",                180,
+         "ZoneIn",                     254
```

Comment:
These need constants on the c++ side.

In general, if want to expose constants to lua, they need to be constants in c++ first.

Reply:
No problem.  Where would you like me to define these in c++?  Here, or in a separate header or class file?